projects/edgedb.com/package.yml

```diff
@@ -0,0 +1,27 @@
+distributable:
+  strip-components: 1
+  url: https://github.com/edgedb/edgedb-cli/archive/refs/tags/v{{version.raw}}.tar.gz
+
+versions:
+  github: edgedb/edgedb-cli/tags
+  strip: /v/
+  
+platforms:
+  - darwin
+  - linux
+
+build:
+  dependencies:
+    rust-lang.org: "1.73.0"
+    rust-lang.org/cargo: "0.74.0"
+    perl.org: "*"
```

Comment:
the specific versions are likely too restrictive. Rust is likely `^1.65`, iirc, and cargo and perl should probably be ^0 and ^5 unless we learn better.